--- archdep.cpp.orig	2008-12-01 12:19:01.000000000 +0100
+++ archdep.cpp	2008-12-01 12:19:50.000000000 +0100
@@ -75,7 +75,7 @@
 int ad_makedirs(char *path)
 {
   strcpy(temp,path);
-  strcat(temp, "/yape");
+  strcat(temp, "/.yape");
   CreateDirectory(temp, NULL);
 
   return 1;
@@ -296,7 +296,7 @@
 	// Possible buffer overflow fixed.
 	strncpy(temp, path, 512);
 	if (strlen(temp) > 506) return 0;
-	strcat(temp, "/yape");
+	strcat(temp, "/.yape");
 	mkdir(temp, 0777);
 
 	return 1;
@@ -307,7 +307,7 @@
 	// Possible buffer overflow fixed.
 	strncpy(temp, tmpchr, 512);
 	if (strlen(temp) > 496) return NULL;
-	strcat( temp, "/yape/yape.conf");
+	strcat( temp, "/.yape/yape.conf");
 	return temp;
 }
 
--- README.SDL.orig	2008-12-01 12:19:59.000000000 +0100
+++ README.SDL	2008-12-01 12:20:34.000000000 +0100
@@ -80,7 +80,7 @@
   LALT + W	: toggle between unlimited speed and 50 Hz frame rate (original speed)
   
   Internal disk LOAD/SAVE operations are supported to the current
-  file system directory, which is usually '/home/<username>/yape'.
+  file system directory, which is usually '/home/<username>/.yape'.
   Any PRG files you may wish to load, should go there, although you can browse the directory tree
   from the user interface, too.
    
